cocos/core/game.ts

```diff
@@ -686,6 +686,14 @@ export class Game extends EventTarget {
                 if (DEBUG) {
                     console.timeEnd('Init Base');
                 }
+
+                if (sys.isXR) {
+                    const xrMSAA = settings.querySettings(Settings.Category.RENDERING, 'msaa') ?? 1;
+                    const xrRenderingScale = settings.querySettings(Settings.Category.RENDERING, 'renderingScale') ?? 1.0;
+                    xr.XrEntry.getInstance().setMultisamplesRTT(xrMSAA);
+                    xr.XrEntry.getInstance().setRenderingScale(xrRenderingScale);
```

Comment:
Could be `xr.entry.setMultiSamplesRTT(xrMSAA)`?

Reply:
This is c++ JSB. Who can I consult to package into TS style?